renovate.json

```diff
@@ -1,4 +1,17 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base"]
+  "extends": [
+    ":dependencyDashboard",
+    ":semanticPrefixFixDepsChoreOthers",
+    "group:monorepos",
+    "group:recommended",
+    "replacements:all",
+    "workarounds:all"
+  ],
+  "packageRules": [
+    {
+      "matchFiles": ["MODULE.bazel"],
+      "enabled": false
```

Comment:
It would be ideal if it could update only the dev_dependency's in this file, of course. Probably not worth the effort.

Reply:
Agreed. I am still researching if this can be done using Renovate configuration. If/when I come up with a better config, I will update this.